core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/WorkspaceMigrationActor.scala

```diff
@@ -1124,8 +1140,11 @@ object WorkspaceMigrationActor {
     }
 
 
-  final def inTransactionT[A](action: DataAccess => ReadWriteAction[Option[A]]): MigrateAction[A] =
-    inTransaction(action).mapF(optT => OptionT(optT.value.map(_.flatten)))
+  final def inTransactionT[A](action: DataAccess => OptionT[ReadWriteAction, A]): MigrateAction[A] =
```

Comment:
Much of the changes in this file are becuase of this line - this allows me to run a transaction over `OptionT` so I don't need to wrap a for-comprehension